match/lib/match/nuke.rb

```diff
@@ -125,7 +124,7 @@ def nuke_it_now!
         UI.success "Successfully deleted profile"
       end
 
-      UI.header "Revoking #{self.certs.count} certificates..." unless self.certs.count == 0
+      UI.header "Revoking #{self.certs.count} certificates..." unless self.certs.count.zero?
```

Comment:
I'd prefer not to have this changed. I assume you're running either the wrong rubocop version, or run it in the wrong directory